tests/jballerina-unit-test/src/test/resources/test-src/statements/whilestatement/while-stmt.bal

```diff
@@ -196,3 +198,39 @@ function assertEquality(any|error expected, any|error actual) {
     string actualValAsString = actual is error ? actual.toString() : actual.toString();
     panic error("AssertionError", message = "expected '" + expectedValAsString + "', found '" + actualValAsString + "'");
 }
+
+function testWhileStatementWithEndlessLoop() returns int {
```

Comment:
We need to call these functions in our unit test `java` classes. 
Similar to this. Otherwise, these are not called.
https://github.com/ballerina-platform/ballerina-lang/blob/c0c8613ceeecfaab27cec95d3cc2832cf3c0be19/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/whilestatement/WhileStmtTest.java#L54